src/main/java/me/mrCookieSlime/Slimefun/api/inventory/BlockMenu.java

```diff
@@ -5,45 +5,46 @@
 import java.nio.file.Files;
 import java.util.logging.Level;
 
+import javax.annotation.Nonnull;
+
+import org.bukkit.Bukkit;
 import org.bukkit.Location;
+import org.bukkit.NamespacedKey;
 import org.bukkit.block.Block;
 import org.bukkit.inventory.ItemStack;
+import org.bukkit.persistence.PersistentDataType;
+import org.bukkit.plugin.Plugin;
+
+import com.jeff_media.customblockdata.CustomBlockData;
```

Comment:
Should probably be shadowed import, not the original? Who knows, Java is strange 🤷